mmv1/products/healthcare/FhirStore.yaml

```diff
@@ -110,6 +110,18 @@ properties:
       - :DSTU2
       - :STU3
       - :R4
+  - !ruby/object:Api::Type::Enum
+    name: complexDataTypeReferenceParsing
+    description: |
+      Enable parsing of references within complex FHIR data types such as Extensions. If this value is set to ENABLED, then features like referential integrity and Bundle reference rewriting apply to all references. If this flag has not been specified the behavior of the FHIR store will not change, references in complex data types will not be parsed. New stores will have this value set to ENABLED after a notification period. Warning: turning on this flag causes processing existing resources to fail if they contain references to non-existent resources.
```

Comment:
I notice this description states `new stores will have this value set to ENABLED after a notification period`. This doesn't appear to be the case as we default this value to `DISABLED`. Should we change the description of remove the default.

If we choose to remove the default will api will return this value regardless of user input. ie: if I don't set this field will it get returned? If so, we would need `default_from_api: true`.

Reply:
Right now this value is default to `Disabled` because our MSA is still pending for approval. After MSA is sent, the default value will be set to `Enabled` after a notification period.

To be more specific, here is the current behaviour:
```
When creating an FHIR store, leaving the value of complexDataTypeReferenceParsing unset, is equivalent to setting this field to DISABLED
```

The new behaviour after the MSA is send & after a notification period:
```
when creating an FHIR store, if the value of complexDataTypeReferenceParsing is left unset, it will automatically be set to ENABLED, and you will not be able to unset this field. By default, all FHIR reference handling functionality, such as enforcing referential integrity and reference rewriting in bundles, will apply to references within complex FHIR data types, such as extensions.
```

Can you elaborate on `if I don't set this field will it get returned`? I am pretty new to Terraform and don't fully understand how everything works.

Reply:
> If user does not set this value, they will use the default value from our API

Correct! You should be able to remove the default value. The issue is, if we leave the default value as is it won't match the intended behavior of the api later. A terraform level default will always send `disabled= true` in an api call regardless as to whether or not it was set.

Reply:
Thanks for clarifying! I set default_from_api to true in a following commit.